README.md

```diff
@@ -17,24 +17,34 @@ This analysis repository presumes that the following python packages are availab
  * dask
  * bottleneck
  * basemap
- * nco with `conda install -c conda-forge nco`
+ * lxml
+ * nco
 
 You can easily install them via the conda command:
 
 ```
-conda install -c conda-forge numpy scipy matplotlib netCDF4 xarray dask bottleneck basemap nco
+conda config --add channels conda-forge
+conda install numpy scipy matplotlib netCDF4 xarray dask bottleneck basemap lxml nco
 ```
 
 ## Running the analysis
   1. Create and empty config file (say `config.myrun`) or copy one of the
      example files in the `configs` directory.
   2. Copy and modify any config options you want to change from 
-     `config.default` into your new config file.  Make sure they have the right
-     section name (e.g. `[run]` or `[output]`).  If nothing esle, you will need
-     to set `baseDirectory` under `[output]` to the folder where output should
-     be stored.  Note: you should not alter `config.default` directly, since
-     this is intended to hold the default configuration that is modified by your
-     new config file.
+     `config.default` into your new config file.  The values form 
+     `config.default` will automatically be used for any options you do not
+     include in your custom config file, so you only need to include options
+     that are different from these defaults.  Make sure the options you copy
+     are in the right section (e.g. `[run]` or `[output]`).  If nothing else,
```

Comment:
Personal preference, but it would be clearer for me for this paragraph to be mostly bullets.  I would suggest after ```into your new config file``` doing something like
 -- **Requirements for custom config files**
      * At minimum you should set `baseDirectory` under `[output]` to the folder where output is stored.  **NOTE** this value should be a unique directory for each run being analyzed.  If multiple runs are analyzed in the same directory, cached results from a previous analysis will not be updated correctly
      * Any options you copy into the config file **must** include the appropriate section header (e.g. '[run]' or '[output]')
      * The entire `config.default` does not need to be used.  This fill will automatically be used for any options you do not include in your custom config file.
      * Given the automatic sourcing of `config.default` you should **not** alter `config.default` directly.

3. run ....